src/components/user/user.service.ts

```diff
@@ -268,6 +268,8 @@ export class UserService {
           canCreateLocation: true,
           canCreateEthnologueLanguage: true,
           canReadEthnologueLanguages: true,
+          canCreatePartner: true,
+          canReadPartners: true,
```

Comment:
I'm not even sure if this is used anymore... @michaelmarshall?

Reply:
oh ya, all those props can be removed.